OWNERS

```diff
@@ -4,3 +4,6 @@ filters:
     - sig-release-reviewers
     approvers:
     - sig-release-approvers
+
+emeritus_approvers:
+- acardace # 2025-07-25
```

Comment:
is it correct to have these here and not in the OWNER_ALIASES?

Reply:
yes, IIUC the [OWNERS emeritus section](https://www.kubernetes.dev/docs/guide/owners/#emeritus)